src/librustc/session/config.rs

```diff
@@ -112,6 +112,26 @@ pub enum OutputType {
     DepInfo,
 }
 
+/// The epoch of the compiler (RFC 2052)
+///
+/// Pronounced e-poach
```

Comment:
I would rather we remove "E-poach" -- I like a funny comment as much as the next person, but I worry that people will actually believe us! (And this is part of rustc's public API)

More importantly, we should comment that the ordering of this enum is significant, and that epochs must go from "early to late" (since there is code that tests e.g., `>= Epoch2018`).